### PR TITLE
fix: release Docker image for Linux

### PR DESCRIPTION
**Pull request by lemeurherve**
Follow-up of https://github.com/jenkins-infra/packer-images/pull/287

Use correct credentials (needs https://github.com/jenkins-infra/kubernetes-management/pull/2644)
Set isPrimary variable as env.BRANCH_IS_PRIMARY doesn't seem to be available in scripted
Add a condition on compute_type for the 'publish docker image' stage

Ref: https://github.com/jenkins-infra/packer-images/issues/282

### CODE REVIEW

File: Jenkinsfile_k8s [hunk 1]
```diff
@@ -1,5 +1,6 @@
-
+def isPrimary = false
```
Comment (dduportal) [inline Jenkinsfile_k8s:1]:
```suggestion
def isBranchPrimary = false
```

WDYt about this name proposal? Rationale is that the qualificative "primary" could be confusing. By adding the name "branch", it aims to make explic when reading the name that "branch" is the subject of the qualificative

**Review by dduportal — APPROVED** [review]
👍

**Review by dduportal — APPROVED** [review]

File: Jenkinsfile_k8s (hunk 1 above)
Comment (smerle33) [reply Jenkinsfile_k8s]: I think I mistakenly clicked a button around here ... sorry ....

Comment (lemeurherve) [reply Jenkinsfile_k8s]: @smerle33 which button? I don't see any kind of modification, should be fine!

**Pull request merged**